### ISSUE TITLE
Fix cert manager hanging problem

### ISSUE THREAD

**Issue by digitalronin** [ministryofjustice/cloud-platform#875]
Cert manager seems to get stuck, sometimes, leading to certificates not
being created/validated until the pod restarts.

In the short term, we should consider creating a cron job which kills the
cert manager pod every so often (e.g. every half hour?) This should avoid
delays setting up certificates.

In the longer term, we need to find out what is causing this issue and fix
it.

### Definition of done

* User certificates are not being held up by cert manager hanging
* We have confidence that cert manager is working correctly, and will continue to do so

**Comment by vijay-veeranki-moj** [comment]
Coudn't reproduce this issue, so closing this issue. 

As discussed with team creating a cron job to cert-manager pod is not needed for now.
